core/src/test/java/com/linecorp/armeria/client/endpoint/healthcheck/HttpHealthCheckedEndpointGroupTest.java

```diff
@@ -4,6 +4,8 @@
 
```

Comment:
Sorry, but could you add the copyright header here as well? (You forgot to add one in your last PR ;-))